cmd/kata-pkgsync/Makefile

```diff
@@ -0,0 +1,21 @@
+TOOL_NAME := kata-pkgsync
+
+VPATH := cli
+
+# Define the C source files by searching in each VPATH element.
+GO_SRCS  := $(foreach dir,$(VPATH),$(wildcard $(dir)/*.go))
+
+ifneq ($(DEBUG),)
+  DEBUG_FLAGS := -gcflags=all="-N -l"
+endif
+
+$(TOOL_NAME): $(GO_SRCS)
+	cd cli && go build $(DEBUG_FLAGS) -o ../$@
```

Comment:
Could you use our usual `git` technique to auto-generate the value of `--version`

- https://github.com/kata-containers/tests/blob/master/cmd/check-markdown/Makefile#L23
- https://github.com/kata-containers/tests/blob/master/cmd/check-markdown/main.go#L29
- https://github.com/kata-containers/tests/blob/master/cmd/check-markdown/main.go#L209

Reply:
I like this very pragmatic instructions :)